src/main/resources/config/application.properties

```diff
@@ -55,7 +55,7 @@ idam.client.secret=${IDAM_OAUTH2_CLIENT_SECRET:QM5RQQ53LZFOSIXJ}
 idam.client.redirect_uri=${IDAM_OAUTH2_REDIRECT_URL:https://localhost:3000/authenticated}
 
 #Document store
-document_management.url=${DOCUMENT_MANAGEMENT_URL:http://dm-store:5005}
+document_management.url=${DOCUMENT_MANAGEMENT_URL:http://localhost:5005}
```

Comment:
Is this related to the suppression fix?

Reply:
no, but it's needed to do any testing with tribunals, as dm-store is not accessible otherwise